library/community/ipfs/values.yaml

```diff
@@ -0,0 +1,29 @@
+image:
+  repository: ipfs/kubo
+  pullPolicy: IfNotPresent
+  tag: v0.18.1
+
+resources:
+  limits:
+    cpu: 4000m
+    memory: 8Gi
+
+ipfs_config:
```

Comment:
Why use snake case when camel case is standard in helm?

Reply:
Will update in a bit

Reply:
fixed